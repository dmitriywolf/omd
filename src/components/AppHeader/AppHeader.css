.app__header {
  padding: 20px 0;
  background-color: rgba(4, 9, 3, 0.91);
}

.app__header-inner {
  display: flex;
  justify-content: space-between;
}
