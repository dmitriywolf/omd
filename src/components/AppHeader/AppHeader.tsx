import React from 'react';
import Menu from '../Menu';
import AppLogo from '../AppLogo';

import './AppHeader.css';

const AppHeader = () => {

  return (
    <header className="app__header">
      <div className="container app__header-inner">
        <AppLogo/>
        <Menu/>
      </div>
    </header>
  )
};

export default AppHeader;
