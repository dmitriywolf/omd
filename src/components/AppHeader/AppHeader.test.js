import React from 'react';
import AppHeader from './AppHeader';
import {shallow} from 'enzyme';
import toJson from "enzyme-to-json";

describe("AppHeader component:", () => {
  it("should render AppHeader component", () => {
    const component = shallow(<AppHeader/>);
    expect(toJson(component)).toMatchSnapshot();
  })
});
