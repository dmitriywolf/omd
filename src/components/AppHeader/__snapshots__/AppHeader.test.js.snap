// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader component: should render AppHeader component 1`] = `
<header
  className="app__header"
>
  <div
    className="container app__header-inner"
  >
    <AppLogo />
    <Menu />
  </div>
</header>
`;
